### DISCUSSION TITLE
OutOfMemoryError: CUDA out of memory  --- AutoMM for Text + Tabular - Quick Start

### DISCUSSION THREAD

**Discussion by AEghasemi** [autogluon/autogluon#4169] [General]
**1) How can I fix the problem? 
2) Is it possible to train it using the CPU?** 

**Code:** 
from autogluon.multimodal import MultiModalPredictor
import uuid

time_limit = 3 * 60    # set to larger value in your applications
model_path = f"./tmp/{uuid.uuid4().hex}-automm_text_book_price_prediction"
predictor = MultiModalPredictor(label='Price', path=model_path)
predictor.fit(train_data, time_limit=time_limit)

**Error:**
OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB. GPU 0 has a total capacty of 3.94 GiB of which 12.94 MiB is free. Including non-PyTorch memory, this process has 3.84 GiB memory in use. Of the allocated memory 3.75 GiB is allocated by PyTorch, and 22.79 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF.

**Comment by joknoll** [comment]
Did you manage to fix it? 
I have the same issue with the code from the the [quick start](https://auto.gluon.ai/stable/tutorials/multimodal/multimodal_prediction/multimodal-quick-start.html) tutorial.
